src/ReachSets/ContinuousPost/TMJets/reach.jl

```diff
@@ -116,7 +124,7 @@ function validated_integ(f!, qq0::AbstractVector{T}, δq0::IntervalBox{N,T},
 
     end
 
-    return view(tv,1:nsteps), view(xv,1:nsteps)
+    return view(tv,1:nsteps), view(xv,1:nsteps), view(xTM1v, :, 1:nsteps)
```

Comment:
Should you not accept this new output here?
https://github.com/JuliaReach/Reachability.jl/blob/1bcaf01f31e14a4d1e68b73a86808f2f2552e955/src/ReachSets/ContinuousPost/TMJets/post.jl#L50-L51

Reply:
Yes, it should read it (i opened https://github.com/JuliaReach/Reachability.jl/issues/635 as follow up) and so far it is ignored.

I didn't do it here because there was no obvious workaround: `ReachSolution` is parametric in `LazySet` and `TaylorModel1` is not a lazy set so i cannot add it to the solution..